worker/uniter/runner/jujuc/secret-get.go

```diff
@@ -91,21 +91,24 @@ func (c *secretGetCommand) Init(args []string) (err error) {
 		}
 		args = args[1:]
 	}
-	if c.metadata {
-		if c.secretUri == nil && c.label == "" {
-			return errors.New("require either a secret URI or label to fetch metadata")
-		}
+
+	if c.secretUri == nil && c.label == "" {
+		return errors.New("require either a secret URI or label")
+	}
+
+	if !c.update {
 		if c.secretUri != nil && c.label != "" {
-			return errors.New("specify either a secret URI or label but not both to fetch metadata")
+			return errors.New("specify either a secret URI or label but not both")
 		}
+	}
```

Comment:
this goes below the metadata check

Reply:
We should only allow `URI` + `--label` for `--update`, right?

Reply:
So after discussed, we allow `secret-get $URI --label label1` to create a consumer label.